src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementDialog.css

```
@@ -0,0 +1,44 @@
+.dialogContainer {
+  padding: 27px 19px 17px 19px;
+
+  & h3 {
```

As I mentioned above, please declare these on they're own new line:
```suggestion
.dialogContainer h3 {
  ...
}
```

Please do the same for the other instances on this page